### DISCUSSION TITLE
What is Photoview business model?

### DISCUSSION THREAD

**Discussion by t18n** [photoview/photoview#713] [Ideas]
I really appreciate the Free software, but I think it is healthy for a project to have a way to make money, which can be used to support the author and software development. Do you have a strategy in mind that you can share?

**Comment by viktorstrate** [comment]
Besides Github sponsors I don't have any plans, do you have any ideas in mind?
Generally I don't want Photoview to become a source of income because it would add a lot of responsibilities.

**Reply by t18n** [reply]
I think there is a lot of potential here, as Photoview has a lot of advantages over the others right now:
- Using FS as album structure
- Android & iOS app, big deal
- Facial recognition
so you can easily turn it to a product, if you want. Of course, it adds more responsibilities, but it will make the product better as well by more people using and more people contributing.

What I am worrying is that, without turning it into something that can generate income, the project will not be able to keep up with its competitors. Some of the ideas:
- Premium features: for example auto-backup, 2FA, etc.
- Cloud hosting
- Crowdfunding
- Patreon
- Opencollective

**Comment by 20pins** [comment]
Just one persons view here, but I see crowdfunding and patreon and just different versions of GitHub sponsors. Maybe I'm missing something. The cloud hosting part seems disconnected from what may be the primary audience, self-hosters who likely already have their own locations for photos. I may be generalizing here too much though. The one items I do find interesting is premium features, but I look at it from a slightly different angle. Prioritizing development based on community interest, like voting. Is there room for a model where someone pays a chunk of money to get a feature in the platform? The risk is it doesn't align with broader community interests, or the core developers ideas on direction for the platform.

My .02

**Comment by viktorstrate** [comment]
> Prioritizing development based on community interest, like voting. Is there room for a model where someone pays a chunk of money to get a feature in the platform? The risk is it doesn't align with broader community interests, or the core developers ideas on direction for the platform.

I think this is a really great way to encourage development, while still being equally free and open for everyone whether they pay or not.

I've connected the project with Issuehunt.io which allows users to add bounties to issues.
https://issuehunt.io/r/photoview/photoview/issues

**Reply by WGandy** [reply]
I'm getting a 401 error at issuehunt when I click on your link. I apparently need to log in??

**Reply by viktorstrate** [reply]
Sorry, try without the last part of the url. I haven't used Issuehunt before so please let me know if I have to do something or if it doesn't work.

https://issuehunt.io/r/photoview/photoview
